support/client/lib/vwf/view/kineticjs.js

```diff
@@ -1742,6 +1753,26 @@ define( [ "module", "vwf/view", "jquery", "vwf/utility", "vwf/utility/color" ],
         viewDriver.kernel.createChild( localDrawingInfo.drawingParentID,
             localDrawingInfo.drawingChildName, drawingDef );
 
+        var action = {
+            type: "drawing",
+            drawingInfo: localDrawingInfo,
+            drawingDef: drawingDef
+        };
+        action.Undo = function(){
```

Comment:
This is just a curiosity question ... was there any specific reason that you chose uppercase letters for these functions?  The js "standard" seems to be to use uppercase letters for constructor functions only (that cues programmers to use the `new` keyword when calling them).